### PR TITLE
DATA-855 Update model json files for grippers

### PR DESCRIPTION
**Pull request by nfranczak**
This PR updates model json files for VG1 and fake gripper.
The geometry for fake gripper is the same as the geometry for VG1.

### CODE REVIEW

**Review by raybjork — APPROVED** [review]

**Comment by biotinker** [comment]
Rather than adding a geometry to the fake gripper, let's remove its model file entirely; that way, users can more easily use it to model any arbitrary gripper.

It only has a model file in the first place, due to some now-obsolete testing that was done back in 2021.

**Review by biotinker — CHANGES_REQUESTED** [review]
Re-posting this comment as actual review feedback.

Rather than adding a geometry to the fake gripper, let's remove its model file entirely; that way, users can more easily use it to model any arbitrary gripper.

It only has a model file in the first place, due to some now-obsolete testing that was done back in 2021.

File: components/gripper/fake/gripper.go [hunk 1]
```diff
@@ -31,12 +28,8 @@ func (config *AttrConfig) Validate(path string) error {
 func init() {
 	registry.RegisterComponent(gripper.Subtype, modelname, registry.Component{
 		Constructor: func(ctx context.Context, _ registry.Dependencies, config config.Component, logger golog.Logger) (interface{}, error) {
-			model, err := referenceframe.UnmarshalModelJSON(gripperjson, "")
-			if err != nil {
-				return nil, err
-			}
 
-			var g gripper.LocalGripper = &Gripper{Name: config.Name, model: model}
+			var g gripper.LocalGripper = &Gripper{Name: config.Name, model: nil}
```
[the remark this answers is not in the document]
Comment (nfranczak) [reply components/gripper/fake/gripper.go:32]: gotcha, will change now

[the remark this answers is not in the document]
Comment (nfranczak) [reply components/gripper/fake/gripper.go:32]: ah, I do not think that is allowed. I am getting:
```
cannot use &(Gripper literal) (value of type *Gripper) as referenceframe.Model value in struct literal: *Gripper does not implement referenceframe.Model (missing method AlmostEquals)compiler[InvalidIfaceAssign](https://pkg.go.dev/golang.org/x/tools/internal/typesinternal#InvalidIfaceAssign)
```

[the remark this answers is not in the document]
Comment (biotinker) [reply components/gripper/fake/gripper.go:32]: Is that from a build or a test?

Also; we should actually be removing the `model` from the `Gripper` struct entirely for `fake`, and returning `nil` explicitly like in other grippers. Example from the softrobotics gripper:

```
// ModelFrame is unimplemented for softGripper.
func (g *softGripper) ModelFrame() referenceframe.Model {
	return nil
}
```

[the remark this answers is not in the document]
Comment (nfranczak) [reply components/gripper/fake/gripper.go:32]: I error showed up right when I pasted `&Gripper{Name: config.Name}` into the model field.
Not an issue anymore as I removed the model field from the fake gripper struct per your request.

**Review by biotinker — APPROVED** [review]
This LGTM, though some changes to some unrelated json files snuck in here; those should be reverted

**Pull request merged**